myportal/soap_service/server_med_rec.php

```diff
@@ -500,62 +500,6 @@ public function arrayToXml($inputArray, $rootElementName = 'root', $childElement
         return $xmlData;
     }
 
-    /**
-    *
-    * @param type $data
-    * @return type
-    */
-    public function getEventLog($data)
```

Comment:
Agree that we shouldn't touch functions in this api directory. These are called by third parties. Also note that the $cols here is not like the other cases, since it is hard-coded right before it is used (ie. not a variable). So, we can leave this alone.

Reply:
Hello @bradymiller am a bit lost. I am expected to revert this change right? Am guessing this is the only change to this PR?

Reply:
hi @prondubuisi ,
Yes, just revert this change and then this PR is ready.
-brady